pallets/automation-time/src/tests.rs

```diff
@@ -267,6 +267,49 @@ fn schedule_xcmp_works() {
 	})
 }
 
+#[test]
+fn schedule_xcmp_through_proxy_works() {
+	new_test_ext(START_BLOCK_TIME).execute_with(|| {
+		let provided_id = vec![50];
+		let alice = AccountId32::new(ALICE);
+		let bob = AccountId32::new(BOB);
```

Comment:
Do not use alice or bob as variable, since they have no functional meaning. If bob is the signer, name the wallet owner or signer.

Reply:
Fixed